extensions/amp-form/0.1/amp-form.js

```diff
@@ -139,9 +141,13 @@ export class AmpForm {
     /** @const @private {!../../../src/service/resources-impl.Resources} */
     this.resources_ = Services.resourcesForDoc(this.form_);
 
-    /** @const @private */
+    /** @const @private {!../../../src/service/viewer-impl.Viewer}  */
     this.viewer_ = Services.viewerForDoc(this.form_);
 
+    /** @private {!../../../src/service/template-renderer-helper.TemplateRendererHelper}*/
+    this.templateRendererHelper_ = new ServerSideTemplateRendererHelper(
```

Comment:
Sorry for being annoying, but this name is sooo long. How about `ServerSideTemplateRenderer` or `SsrTemplateHelper`?